src/parsers/css/codeExample.txt

```diff
@@ -0,0 +1,11 @@
+h1 {
+    background-color: green;
+}
+
+div {
+    background-color: lightblue;
+}
+
+p {
+    background-color: yellow;
+}
```

Comment:
I know it's WIP so probably just wanted to get something working.

Might be good to leave the same kind of comments from the js example and maybe show off some more css ast nodes?